core/services/ping/main.py

```diff
@@ -2,16 +2,56 @@
 import argparse
 import asyncio
 import logging
+from typing import Any, List
+
+from commonwealth.utils.apis import PrettyJSONResponse
+from commonwealth.utils.logs import InterceptHandler
+from fastapi import FastAPI
+from fastapi.responses import HTMLResponse
+from fastapi_versioning import VersionedFastAPI, version
+from loguru import logger
+from typedefs import PingDeviceDescriptorModel
+from uvicorn import Config, Server
 
 from pingmanager import PingManager
 from pingprober import PingProber
 from portwatcher import PortWatcher
 
+app = FastAPI(
+    title="Ping Manager API",
+    description="Ping Manager is responsible for managing Ping devices connected to Companion.",
+    default_response_class=PrettyJSONResponse,
+    debug=True,
+)
+logger.info("Starting Ping Service.")
+
+# TODO: move to singleton
+ping_manager = PingManager()
+
+
+@app.get("/sensors", response_model=List[PingDeviceDescriptorModel], summary="Current sensors detected.")
+@version(1, 0)
+def get_sensors() -> Any:
+    return [PingDeviceDescriptorModel.from_descriptor(device) for device in ping_manager.devices()]
```

Comment:
Maybe we should log this devices list, to make debugging easier?

Reply:
Done.